### DISCUSSION TITLE
INSERT mode when creating/entering CodeCompanion chat

### DISCUSSION THREAD

**Discussion by S1M0N38** [olimorris/codecompanion.nvim#222] [Ideas]
When I first enter an empty chat, there is only one thing you can do:
INSERT mode and start typing.

I think this should be the default behavior

**Comment by olimorris** [comment]
It's something I've thought about too. My reason for not making it the default was so new users would see the virtual text and know they can access the info menu with `?`...but I think when you're toggling a chat buffer, it should go into insert mode right away

**Reply by olimorris** [reply]
@S1M0N38 this is now in v.3.2.0.

Would you expect this to be the case when you're toggling the chat buffer too?

**Reply by S1M0N38** [reply]
I don’t think so. When switching windows in Neovim, you usually remain in normal mode. I believe the empty chat is an exception.
